resolverway/views.py

```diff
@@ -106,16 +106,23 @@ def set_user_info(self, request):
             self.real_ip = request.headers.get('x-real-ip', None)
         session = request.cookies.get('session', None)
         if session:
-            account = redis_db.get(session)
+            try:
+                account = redis_db.get(name=current_app.config['REDIS_NAME_PREFIX']+session)
+            except:
+                account = None
+                current_app.logger.error('exception on getting user info from cache with session=%s' % (session))
```

Comment:
If we do `except e:`, then we could add in the error message what was the exception: `current_app.logger.error('exception on getting user info from cache with session=%s: %s' % (session, str(e))`. This will help us investigate exceptions in the future, since all the info will be in the logs.